eventnotifier/types/types.go

```diff
@@ -5,8 +5,11 @@
 
 package types
 
+import "sigs.k8s.io/controller-runtime/pkg/client"
+
 type NotifiersCommonConfig struct {
 	ClusterName string
+	Client      client.Client
```

Comment:
nit: consider using pointer to avoid duplicating `client` object

```suggestion
	Client      *client.Client
```